agent/bench-scripts/pbench-fio

```diff
@@ -449,6 +460,12 @@ function fio_process_options() {
 			;;
 		esac
 	done
+
+	if [[ "${install_only}" == "y" ]]; then
+		fio_local_check_install
+		exit ${?}
+	fi
```

Comment:
What confuses me a bit here is what seems to be an implicit assumption (based on collating the problem description and code changes) that `${install_only}` means "agent" vs "controller". Is that really guaranteed? Is there a way the environment dependency/assumption could be more explicit and robust?

Reply:
It really doesn't matter much, from what I can tell.

Right now, the only use of `--install` is from `pbench-fio` when it remotely invokes itself on the list of clients.

If somebody adds `--install` then `pbench-fio` will just check locally an exit immediately.

I have removed it from the `usage` output.